src/main/groovy/com/google/protobuf/gradle/ProtobufExtension.groovy

```diff
@@ -80,13 +79,21 @@ abstract class ProtobufExtension {
   }
 
   String getGeneratedFilesBaseDir() {
```

Comment:
Deprecated too? It's better to use getGeneratedFilesBaseDirProperty directly

Reply:
getGeneratedFilesBaseDirProperty is PackageScope; we want to get rid of people changing this setting entirely. If this is to be deprecated, that'd be so that it is just completely unavailable in the future. I could be okay with doing that, but it didn't seem all that important right now.

Reply:
Ok.